{"report": {"environment": {"Python": "3.11.5", "Platform": "macOS-13.3.1-arm64-arm-64bit"}, "tests": [{"name": "test_API_automation.py::test_post_request_validation", "duration": 0.6344227069057524, "run_index": 0, "setup": {"name": "setup", "duration": 0.316686290781945, "outcome": "passed", "stderr": "DEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.address`.\nDEBUG:faker.factory:Provider `faker.providers.address` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.automotive`.\nDEBUG:faker.factory:Provider `faker.providers.automotive` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.bank`.\nDEBUG:faker.factory:Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.barcode`.\nDEBUG:faker.factory:Provider `faker.providers.barcode` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.color`.\nDEBUG:faker.factory:Provider `faker.providers.color` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.company`.\nDEBUG:faker.factory:Provider `faker.providers.company` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.credit_card`.\nDEBUG:faker.factory:Provider `faker.providers.credit_card` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.currency`.\nDEBUG:faker.factory:Provider `faker.providers.currency` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.date_time`.\nDEBUG:faker.factory:Provider `faker.providers.date_time` has been localized to `en_US`.\nDEBUG:faker.factory:Provider `faker.providers.emoji` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:faker.factory:Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.geo`.\nDEBUG:faker.factory:Provider `faker.providers.geo` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.internet`.\nDEBUG:faker.factory:Provider `faker.providers.internet` has been localized to `en_US`.\nDEBUG:faker.factory:Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.job`.\nDEBUG:faker.factory:Provider `faker.providers.job` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.lorem`.\nDEBUG:faker.factory:Provider `faker.providers.lorem` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.misc`.\nDEBUG:faker.factory:Provider `faker.providers.misc` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.passport`.\nDEBUG:faker.factory:Provider `faker.providers.passport` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.person`.\nDEBUG:faker.factory:Provider `faker.providers.person` has been localized to `en_US`.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.phone_number`.\nDEBUG:faker.factory:Provider `faker.providers.phone_number` has been localized to `en_US`.\nDEBUG:faker.factory:Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:faker.factory:Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:faker.factory:Provider `faker.providers.sbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:faker.factory:Looking for locale `en_US` in provider `faker.providers.ssn`.\nDEBUG:faker.factory:Provider `faker.providers.ssn` has been localized to `en_US`.\nDEBUG:faker.factory:Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): gorest.co.in:443\nDEBUG:urllib3.connectionpool:https://gorest.co.in:443 \"POST /public/v2/posts HTTP/1.1\" 422 None\nDEBUG:conftest:POST request to https://gorest.co.in/public/v2/posts\nDEBUG:conftest:Headers: {'Accept': 'application/json', 'Content-Type': 'application/json', 'Authorization': 'Bearer a6feb1e9909e836edaa64ebb91544d694441cb09f1bc0940fe3f11f568aa191f'}\nDEBUG:conftest:Request data: {}\nDEBUG:conftest:Response status code: 422\nDEBUG:conftest:Response data: [{\"field\":\"user\",\"message\":\"must exist\"},{\"field\":\"user_id\",\"message\":\"is not a number\"},{\"field\":\"title\",\"message\":\"can't be blank\"},{\"field\":\"body\",\"message\":\"can't be blank\"}]\n", "log": "DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.address`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.address` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.automotive`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.automotive` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.bank`.\nDEBUG    faker.factory:factory.py:86 Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.barcode`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.barcode` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.color`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.color` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.company`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.company` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.credit_card`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.credit_card` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.currency`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.currency` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.date_time`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.date_time` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.emoji` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.geo`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.geo` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.internet`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.internet` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.job`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.job` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.lorem`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.lorem` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.misc`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.misc` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.passport`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.passport` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.person`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.person` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.phone_number`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.phone_number` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.sbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.ssn`.\nDEBUG    faker.factory:factory.py:95 Provider `faker.providers.ssn` has been localized to `en_US`.\nDEBUG    faker.factory:factory.py:106 Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.\nDEBUG    urllib3.connectionpool:connectionpool.py:1014 Starting new HTTPS connection (1): gorest.co.in:443\nDEBUG    urllib3.connectionpool:connectionpool.py:473 https://gorest.co.in:443 \"POST /public/v2/posts HTTP/1.1\" 422 None\nDEBUG    conftest:conftest.py:266 POST request to https://gorest.co.in/public/v2/posts\nDEBUG    conftest:conftest.py:267 Headers: {'Accept': 'application/json', 'Content-Type': 'application/json', 'Authorization': 'Bearer a6feb1e9909e836edaa64ebb91544d694441cb09f1bc0940fe3f11f568aa191f'}\nDEBUG    conftest:conftest.py:268 Request data: {}\nDEBUG    conftest:conftest.py:269 Response status code: 422\nDEBUG    conftest:conftest.py:270 Response data: [{\"field\":\"user\",\"message\":\"must exist\"},{\"field\":\"user_id\",\"message\":\"is not a number\"},{\"field\":\"title\",\"message\":\"can't be blank\"},{\"field\":\"body\",\"message\":\"can't be blank\"}]"}, "call": {"name": "call", "duration": 0.0004912917502224445, "outcome": "passed"}, "teardown": {"name": "teardown", "duration": 0.0005588335916399956, "outcome": "passed"}, "outcome": "passed"}], "summary": {"passed": 1, "num_tests": 1, "duration": 0.34369802474975586}, "created_at": "2023-09-26 10:19:26.625627"}}